app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new(comment_params)
+    @comment.user_id = current_user.id
+    if @comment.save
+      flash[:notice] = "Comment was successfully created."
+      redirect_to [@topic, @post]
+    else
+      flash[:error] = "There was an error creating the comment. Please try again."
+      redirect_to [@topic, @post]
+    end
+  end
```

Comment:
How could lines 6-12 be modified so the `redirect_to [@topic, @post]` line is not repeated?